projects/angular/src/accordion/accordion-panel.html

```diff
@@ -41,7 +41,10 @@
         (@toggle.done)="collapsePanelOnAnimationDone(panel)"
         class="clr-accordion-content"
       >
-        <div class="clr-accordion-inner-content">
+        <div
+          class="clr-accordion-inner-content"
+          [class.clr-accordion-inner-content-disabled]="isAccordion && panel.disabled"
```

Comment:
I don't see the inner content disabled state specified in figma, can you verify with design that we do in fact want the content panel to render a disabled color. If so, we should set the styling of the inner content from the clr-accordion-panel-disabled class instead of adding a new one here.